### DISCUSSION TITLE
Add ENV sample

### DISCUSSION THREAD

**Discussion by grtsnx** [TafadzwaD/artistaa-whatsapp-chatbot#4] [General]
it will be good to have a .env.sample file so anyone forking this project can know the exact variable names used in the project. 

.env.sample
WHATSAPP_CLOUD_API_WEBHOOK_VERIFICATION_TOKEN=
HASHING_SALT=
OPENAI_API_KEY=
REDIS_URL=upstash.io
WHATSAPP_CLOUD_API_ACCESS_TOKEN=
WHATSAPP_CLOUD_API_PHONE_NUMBER_ID=
WHATSAPP_CLOUD_API_VERSION=v21.0

**Comment by TafadzwaD** [comment]
Done! Thanks for the suggestion @thegrtnx

**Reply by grtsnx** [reply]
Welldone and thank you @TafadzwaD
